From dd67e78c0dd57f8081c8f3165ce6b0cfd0d80643 Mon Sep 17 00:00:00 2001
From: Timo Rothenpieler <timo.rothenpieler@uni-bremen.de>
Date: Fri, 27 Aug 2021 15:48:58 +0200
Subject: [PATCH] Fix build without netloc

---
 auxdir/x_ac_netloc.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/auxdir/x_ac_netloc.m4 b/auxdir/x_ac_netloc.m4
index 501cfab3d4..de54e28979 100644
--- a/auxdir/x_ac_netloc.m4
+++ b/auxdir/x_ac_netloc.m4
@@ -21,7 +21,7 @@ AC_DEFUN([X_AC_NETLOC],
     [AS_IF([test "x$with_netloc" != xno && test "x$with_netloc" != xyes],
            [_x_ac_netloc_dirs="$with_netloc"])])
 
-  if [test "$with_netloc" = xno]; then
+  if [test "x$with_netloc" = xno]; then
     AC_MSG_WARN([support for netloc disabled])
   else
     AC_CACHE_CHECK(
-- 
2.31.1

